articles/cognitive-services/Face/face-how-to-install-containers.md

```diff
@@ -64,13 +64,13 @@ There are several ways to authenticate with the private container registry for C
 
 Use the [docker login](https://docs.docker.com/engine/reference/commandline/login/) command, as shown in the following example, to log into `containerpreview.azurecr.io`, the private container registry for Cognitive Services Containers. Replace *\<username\>* with the user name and *\<password\>* with the password provided in the credentials you received from the Azure Cognitive Services team.
 
-```docker
+```DockerFile
```

Comment:
note: this still probably isn't a correct language block because this is actually a block of commands